### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by movermeyer**
More documentation for the code as it exists today.

### CODE REVIEW

**Comment by movermeyer** [comment]
Note: The CircleCI checks doesn't seem to running.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+            - [Documentation](#documentation)
+            - [Pull Requests](#pull-requests)
+
+## I don't want to read this whole thing I just have a question!!!
+
+Sure. First [search the existing issues](https://github.com/exactEarth/ProgressTracker/issues?utf8=%E2%9C%93&q=is%3Aissue) to see if one of the existing issues answers it. If not, simply [create an issue](https://github.com/exactEarth/ProgressTracker/issues/new) and ask your question.
+
+## Design Philosophy
+
+progress_tracker's goal is to be the fastest ISO 8601 parser available for Python. It probably will never support the complete grammar of ISO 8601, but it will be correct for the chosen subset of the grammar. It will also be robust against non-conforming inputs. Beyond that, performance is king.
```
Comment (NickG123) [inline CONTRIBUTING.md]: 🤔

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+* Set the `"editor.formatOnSave": true` setting to automatically auto-format the code on saves.
+* Set the `"python.linting.mypyEnabled": true` setting to automatically run `mypy` on the code on saves.
+
+#### Supported Python Versions
+
+progress_tracker is built with, and tested against, cPython versions 3.6+ (for the full list see the [README](README.rst)). Please make sure that you do not accidentally make use of features that are specific to certain versions of Python. Feel free to make use of modern features of the languages, but you also need to provide mechanisms to support the other versions as well.
+
+#### Supported Operating Systems
+
+progress_tracker supports running on multiple operating systems, including Windows. Make sure to test changes on both a Windows (MSVC) and Linux (gcc) machine to ensure compatibility.
```
Comment (NickG123) [inline CONTRIBUTING.md]: I don't think MVSC or gcc is required here

File: CONTRIBUTING.md (hunk 1 above)
Comment (movermeyer) [reply CONTRIBUTING.md]: Right...

File: CONTRIBUTING.md (hunk 2 above)
Comment (movermeyer) [reply CONTRIBUTING.md]: Right...

**Comment by NickG123** [comment]
CircleCI should be working now I think.  Might need a new push first tho

**Pull request merged**